example/example.c

```diff
@@ -182,6 +182,6 @@ int main(void) {
 int report_error(void) {
     char message[SIR_MAXERROR] = {0};
     uint16_t code              = sir_geterror(message);
-    fprintf(stderr, "libsir error: (%"PRIu16", %s)\n", code, message);
+    fprintf(stderr, "libsir error: (%" PRIu16", %s)\n", code, message);
```

Comment:
Don't you need a space on both sides if you need spaces?

Reply:
It only cares about the leading spaces, oddly.  Cppreference explains the dumb rationale somewhere, I think at where I linked it, if you care to know.  Hah.